helm/charts/infra/templates/connector/configmap.yaml

```diff
@@ -7,35 +7,30 @@ metadata:
 {{- include "connector.labels" . | nindent 4 }}
 data:
   infra.yaml: |
-{{- range $key, $val := omit .Values.connector.config "accessKey" }}
-{{- if kindIs "invalid" $val }}
-    # skipping invalid value: {{ $val }} ({{ kindOf $val }})
-{{- else if kindIs "map" $val }}
-    {{ $key }}:
-{{- $val | toYaml | nindent 6 }}
-{{- else if kindIs "slice" $val }}
-    {{ $key }}:
-{{- $val | toYaml | nindent 6 }}
-{{- else if kindIs "string" $val }}
-    {{ $key }}: {{ tpl $val $ }}
-{{- else }}
-    {{ $key }}: {{ $val }}
-{{- end }}
-{{- end }}
```

Comment:
This generic "import all config" doesn't work anymore now that `server` changed from a string to a struct.  We already have explicit handling of all the options, so we don't need this I think. Users can always inject specific config settings using env vars, which I think works better here because it doesn't require type checking the value.